uaclient/cli.py

```diff
@@ -1297,16 +1297,17 @@ def action_enable(args, *, cfg, **kwargs):
         names, cfg
     )
     ret = True
-    for ent_name in entitlements_found:
+    for ent_name, ent_ret, reason, e in actions.enable_entitlements_by_name(
+        cfg,
+        entitlements_found,
+        assume_yes=args.assume_yes,
+        allow_beta=args.beta,
+        access_only=access_only,
+        variant=variant,
+    ):
         try:
-            ent_ret, reason = actions.enable_entitlement_by_name(
-                cfg,
-                ent_name,
-                assume_yes=args.assume_yes,
-                allow_beta=args.beta,
-                access_only=access_only,
-                variant=variant,
-            )
+            if e is not None:
```

Comment:
Is there a reason why we shouldn't raise the exception directly on the `enable_entitlements_by_name` function ?